R/miaTime.R

```diff
@@ -138,3 +138,50 @@ NULL
 #' @keywords data
 #'
 NULL
+
+#' @title CrestStudyData
```

Comment:
From where this name comes from?

Reply:
Could be also according to the SurnameYear of the publication (based on first author's name).